src/main/java/org/swisspush/reststorage/FileSystemRestStorageRunner.java

```diff
@@ -9,10 +9,10 @@
  * Deploys the rest-storage to vert.x.
  * Used in the standalone scenario.
  */
-public class RestStorageRunner {
+public class FileSystemRestStorageRunner {
 
     public static void main(String[] args) {
-        Vertx.vertx().deployVerticle("org.swisspush.reststorage.RestStorageMod", event ->
-                LoggerFactory.getLogger(RestStorageMod.class).info("rest-storage started"));
+        Vertx.vertx().deployVerticle(new RestStorageMod(), event ->
```

Comment:
```patch
-    vertx.deployVerticle(new RestStorageMod(), ...);
+    vertx.deployVerticle(RestStorageMod.class.getName(), ...);
```

In fact I meant to use [java.lang.Class#getName()](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Class.html#getName()). I didn't even know that we also can pass an instance there :D

So if you're sure that this is a good idea, ok for me.

Still I assume its recommended to pass a [FQDN](https://docs.oracle.com/javase/specs/jls/se17/html/jls-6.html#jls-6.7). My intent behind suggesting to use "getName()" was that we can get rid of the magic string. Which in turn also makes finding usages much simpler.

Reply:
The _RedisRestStorageRunner_ and _FileSystemRestStorageRunner_ classes are just used for local standalone running. Mostly this is the case during development. We also work with passing instances of the Verticle in the unit tests already.